napari/_qt/qt_viewer.py

```diff
@@ -136,35 +156,40 @@ def screenshot(self, region=None, size=None, bgcolor=None):
         """
         return self.canvas.render(region, size, bgcolor)
 
-    def _open_images(self):
-        """Adds image files from the menubar."""
+    def _open_files(self):
+        """Adds files from the menubar."""
         filenames, _ = QFileDialog.getOpenFileNames(
             parent=self,
-            caption='Select image(s)...',
+            caption='Select image(s).../volume(npy or npz files)',
             directory=self._last_visited_dir,  # home dir by default
         )
         self._add_files(filenames)
 
     def _add_files(self, filenames):
-        """Adds an image layer to the viewer.
+        """Adds an image/volume layer to the viewer.
 
         Whether the image is multichannel is determined by
         :func:`napari.util.misc.is_multichannel`.
 
-        If multiple images are selected, they are stacked along the 0th
-        axis.
-
         Parameters
         -------
         filenames : list
             List of filenames to be opened
         """
-        if len(filenames) > 0:
-            image = read(filenames)
+        image = read(filenames)
+        if len(filenames) == 1:
             self.viewer.add_image(
                 image, multichannel=is_multichannel(image.shape)
             )
-            self._last_visited_dir = os.path.dirname(filenames[0])
+        elif len(filenames) > 1:
```

Comment:
@sofroniewn should I revert these changes too?

Reply:
yes please

Reply:
reverted